scripts/prepare-unity.sh

```diff
@@ -37,7 +37,24 @@ export GOPATH
 
 if [ -d "$IMPROBABLE_UNITY_VERSIONED" ]; then
   echo "Unity is already installed in:" $IMPROBABLE_UNITY_VERSIONED
-  exit 0
+
+  if isWindows; then
+    PLAYBACK_ENGINE_ROOT="$IMPROBABLE_UNITY_VERSIONED/Editor/Data/PlaybackEngines"
+  elif isMacOS; then
+    PLAYBACK_ENGINE_ROOT="$IMPROBABLE_UNITY_VERSIONED/PlaybackEngines"
+  fi
+
+  if [ -d "$PLAYBACK_ENGINE_ROOT/LinuxStandaloneSupport" ] && \
+     [ -d "$PLAYBACK_ENGINE_ROOT/AndroidPlayer" ] && \
+     [ -d "$PLAYBACK_ENGINE_ROOT/iOSSupport" ] && \
+     [ -d "$PLAYBACK_ENGINE_ROOT/WindowsStandaloneSupport" ] && \
+     [ isMacOS || -d "$PLAYBACK_ENGINE_ROOT/MacStandaloneSupport" ]
```

Comment:
Why does Mac gotta be a special snowflake

Reply:
Mac has it's own support installed in the Unity app package itself instead of standalone playback engine

Reply:
A comment to that effect would be very useful.

Reply:
will add that 👍